### PR TITLE
fix warning in example_test.py

### PR DESCRIPTION
**Pull request by karolinavelkaja**
Signed-off-by: Karolina Rogowska <karolina.rogowska@intel.com>

### CODE REVIEW

File: test/functional/tests/example/example_test.py [hunk 1]
```diff
@@ -16,7 +16,10 @@
 
 
 def setup_module():
-    TestRun.LOGGER.warning("Entering setup method")
+    """
+    Function called by python
```
Comment (Ostrokrzew) [inline test/functional/tests/example/example_test.py:20]: *Python
This function is used nowhere anyway, so IMO can be dropped.

**Review by Ostrokrzew — CHANGES_REQUESTED** [review]

File: test/functional/tests/example/example_test.py (hunk 1 above)
Comment (karolinavelkaja) [reply test/functional/tests/example/example_test.py:20]: I thought so but during debug turned out that it is called by python.py

Comment (karolinavelkaja) [reply test/functional/tests/example/example_test.py:20]: so no - it should not be dropped and that's why the description had been added here.

**Review by Ostrokrzew — APPROVED** [review]

**Comment by katlapinka** [comment]
retest this please

**Review by katlapinka — APPROVED** [review]

**Comment by ArkadiuszNeumann** [comment]
retest this please

**Pull request merged**